liquidator/liquidator.js

```diff
@@ -36,29 +36,23 @@ class Liquidator {
     // Get the latest undercollateralized positions from the client.
     const underCollateralizedPositions = this.empClient.getUnderCollateralizedPositions(priceFeed);
 
-    if (underCollateralizedPositions.length > 0) {
-      for (const underCollateralizedPosition of underCollateralizedPositions) {
-        Logger.info({
-          at: "liquidator",
-          message: "undercollateralized positions detected 🚨",
-          underCollateralizedPosition: underCollateralizedPosition
-        });
-      }
-    } else {
+    if (underCollateralizedPositions.length === 0) {
       Logger.debug({
         at: "liquidator",
         message: "No undercollateralized position"
       });
+      return;
     }
 
     let liquidationPromises = []; // store all promises to resolve in parallel later on.
     for (const position of underCollateralizedPositions) {
       // TODO: add additional information about this liquidation event to the log.
       Logger.info({
         at: "liquidator",
-        message: "liquidating sponsor 🔥",
+        message: "liquidating sponsor🔥",
```

Comment:
These emojis are not gonna age well when printing to console IMO. Wouldn't it just print as ":fire:" in bare-bones terminals? I'm imagining the AWS server we have running this client.

Reply:
I'm not super opinionated about styling of text. However, @chrismaree made the suggestions to change/add these, so I think we should come to a decision about whether to have them or not and be consistent about it.

Reply:
I'm going to leave this as is since the emojis predated this PR. Definitely discuss with @chrismaree, and feel free to open a PR to remove if y'all decide that's the right thing to do.